server/trillian_db_client/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8
+
+RUN apt-get update
+RUN apt-get install -y mysql-client
```

Comment:
It is important to do `apt-get update` and `apt-get install` as a single `RUN` command, otherwise you can get screwed over by Docker layer caching. See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run.